src/metatensor/models/experimental/pet/__init__.py

```diff
@@ -1,7 +1,11 @@
 from .model import Model, DEFAULT_HYPERS  # noqa: F401
 from .train import train  # noqa: F401
+import torch
 
-DEVICES = ["cuda"]
+__ARCHITECTURE_CAPABILITIES__ = {
+    "supported_devices": ["cpu"],
```

Comment:
```suggestion
    "supported_devices": ["cuda"],
```
PET can only be trained on CUDA (for now)

Reply:
**can only** or **should only** be trained on CUDA? 

@abmazitov is running PET on CPU with LAMMPS (until I write the code to use other devices), so I would be surprised if one can not train PET on CUDA at all (although it would likely be very slow ^_^)

Reply:
Training only works on CUDA for now (hardcoded inside PET). Once it's exported, moving to CPU works

Reply:
👍